olp-cpp-sdk-core/tests/client/ApiLookupClientImplTest.cpp

```diff
@@ -393,6 +393,69 @@ TEST_F(ApiLookupClientImplTest, LookupApiCustomProvider) {
   testing::Mock::VerifyAndClearExpectations(cache_.get());
 }
 
+TEST_F(ApiLookupClientImplTest, LookupApiCustomCatalogProvider) {
```

Comment:
Please find a smaller name. ApiLookupClient only has two public methods and both are called `LookupApi` so I dont think you need to especially mention that.